tests/steps/application_v2_steps.py

```diff
@@ -40,24 +43,49 @@ def operation_string_to_enum(operation):
         )
 
 
+# Takes in a tuple where first element is the encoding and second element is value.
+# If there is only one element, then it is assumed to be an int.
+def process_app_args(sub_arg):
```

Comment:
```suggestion
def process_app_arg(sub_arg):
```
Should this be renamed in the singular without an "s" because _if I understand correctly_ only one arg is being processed?